### ISSUE TITLE
Is there a mistake in your schematic? MISO and MOSI are NC

### ISSUE THREAD

**Issue by DeflateAwning** [JackGruber/Arduino-Pro-Mini-LoRa-Sensor-Node#6]
I would expect that MISO and MOSI need to be connected for this system to work.

**Comment by stkdiretto** [comment]
That is correct. The schematics of the board appear to be correct, as opposed to the circuit diagram.

Connections are:

| Arduino | RFM95 |
| --- | --- |
| GND | GND |
| 5 | DIO0 |
| 6 | DIO1 |
| 7 | DIO2 |
| 10 | NSS |
| 11 | MOSI |
| 12 | MISO |
| 13 | SCK |
| A0 | RST |
| VCC | 3V3 |
| GND | GND |
